### PR TITLE
Add release note for 1.11

### PR DESCRIPTION
**Pull request by fisherxu**
Signed-off-by: fisherxu <xufei40@huawei.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
Add release note for 1.11 release

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.11.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+New version of the lightweight engine Edged, optimized from kubelet and integrated in edgecore, move to Beta.
+New Edged will still communicate with the cloud through the reliable transmission tunnel.
+
+Refer to the links for more details.
+(Dev-Branch for beta: [feature-new-edged](https://github.com/kubeedge/kubeedge/tree/feature-new-edged))
+
+
+## Important Steps before Upgrading
+
+If you want to use keadm to deploy the KubeEdge v1.11.0, please note that the keadm `init` and `join` have been replaced by `beta init` and `beta join`.
```
Comment (gy95) [inline CHANGELOG/CHANGELOG-1.11.md:88]: If you want to use keadm to deploy the KubeEdge v1.11.0, please note that the keadm `beta init` and `beta join` have been replaced by `init` and `join`.

File: CHANGELOG/CHANGELOG-1.11.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Refer to the links for more details.
+([#70](https://github.com/kubeedge/mappers-go/pull/70))
+
+
+
+### Beta sub-commands in Keadm to GA
+
+Some new sub-commands in keadm move to GA, including containerized deployment, offline installation, etc.
+Original `init` and `join` are replaced by `beta init` and `beta join`.
```
Comment (gy95) [inline CHANGELOG/CHANGELOG-1.11.md:56]: Original `beta init` and `beta join` are replaced by `init` and `join`.

**Comment by gy95** [comment]
LGTM

**Review by kevin-wangzefeng — COMMENTED** [review]
Minor comments

**Review by vincentgoat — COMMENTED** [review]
Should we include PR 3899 for the panic issue?

**Comment by fisherxu** [comment]
> Should we include PR 3899 for the panic issue?

Will in follow up PR by @gy95

**Pull request merged**